src/partials/id_verification_required.hbs

```diff
@@ -27,12 +33,9 @@
           {{verificationCode}}
         </div>
       </div>
-      <button id="copy" class="button button--tertiary">Copy</button>
     </div>
     <div class="text-block">
-      <div class="text-block--overflow-wrap">
-        <a tabindex="0" data-actionId="cancelAuthentication" target="_self" class="anchor">Cancel</a>
-      </div>
+      <a tabindex="0" id="retryVerification" target="_self" class="anchor">Cancel</a>
```

Comment:
Does the new adapter still support cancelAuthentication? This looks like it could be a breaking change

Reply:
The new adapter still supports cancelAuthentication - by clicking cancel in the first screen or the timeout screen. All other screens go to the first screen on cancel click.